src/main/java/org/konstructs/forest/ActorManager.java

```diff
@@ -0,0 +1,97 @@
+package org.konstructs.forest;
```

Comment:
@nsg I think this base class looks very promising. I like the implemented onX methods, that means that one must not fully understand akka, while at the same time one must not implement the methods. Really nice, but one small thing can be improved. I think they should all call `unhandled(message);`, that way the message is also logged.